apps/app/components/Event/index.js

```diff
@@ -122,9 +122,14 @@ const Event = ({
               <EventInfo {...event} />
             )}
             {role === EUser.Organizer ? (
-              <Link href={`/admin/event/${event.id}`}>
-                <Button type="primary">Info</Button>
-              </Link>
+              <>
+                <Link href={`/admin/event/${event.id}`}>
+                  <Button type="primary">Info</Button>
+                </Link>
+                <Link href={`/admin/lectures/new`}>
+                  <Button type="primary">Criar Lecture</Button>
```

Comment:
This button only makes sense if the new lecture page can receive a prop called, for instance, `event_id`. Otherwise, it's just a general redirect and you can remove it.